script.js

```diff
@@ -1,10 +1,18 @@
-document.addEventListener('DOMContentLoaded', () => {
-    const calendarCells = document.getElementById('calendarCells');
-    const hoursInDay = 19;
-    const daysInWeek = 7;
-    for (let i = 0; i < hoursInDay * daysInWeek; i++) {
-        const cell = document.createElement('div');
-        cell.className = 'cell';
-        calendarCells.appendChild(cell);
-    }
-});
\ No newline at end of file
+function fillCells() {
+    document.addEventListener('DOMContentLoaded', () => {
+        const calendarCells = document.getElementById('calendarCells');
+        const cellsInOneColumn= 19;
+        const daysInWeek = 7;
+        for (let i = 0; i < cellsInOneColumn * daysInWeek; i++) {
+            const cell = document.createElement('div');
+            cell.className = 'cell';
+            calendarCells.appendChild(cell);
+        }
+    });
+}
+
+function init() {
+    fillCells();
```

Comment:
```suggestion
    renderCalendarCells();
```
because function is not filling cells with anything, it creates divs with certain class and appends them to DOM